pages/linux/phpdismod.md

```diff
@@ -1,6 +1,7 @@
 # phpdismod
 
 > Disable PHP extensions on Debian-based OSes.
+> More information: https://linuxcommandlibrary.com/man/phpdismod
```

Comment:
```suggestion
> More information: <https://salsa.debian.org/php-team/php-defaults>.
```

Reply:
Thank you.
I have a question about the website: which website is legit? How to be sure?

Reply:
Additional question, `phpdismod` is not so easy to find on the salsa.debian.org website, is it okay?

Reply:
I reviewed the source code that Debian use to build the package that contains this command, but if there is no man page available, I think the source code is the best that we can provide. `linuxcommandlibrary` doesn't add more information neither. :man_shrugging:

Reply:
🆗  Thank you!